fontconfig/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "fontconfig"
+version = "0.1.0"
+authors = [
+    "Austin Bonander <austin.bonander@gmail.com>",
+    "Manuel Reinhardt <manuel.jr16@gmail.com>",
+    "YesLogic Pty. Ltd. <info@yeslogic.com>"
+]
+license = "MIT"
+keywords = ["font", "search", "wrapper"]
+repository = "https://github.com/yeslogic/fontconfig-rs"
+homepage = "https://github.com/yeslogic/fontconfig-rs"
```

Comment:
Ahh these changes were made assuming I'd publish this as a new crate. Can change.